poglink/cogs/rates.py

```diff
@@ -175,7 +135,7 @@ async def compare_and_notify_all(self, **kwargs):
                             )
                             embed_description = stable_diff.to_embed()
                             await self.send_embed(
-                                embed_description, url, title=kwargs.get("embed_title")
+                                embed_description, rates_dict={"url": url}
```

Comment:
Honestly I can't bring myself to understand the diff-checking logic above this, and this was a bit of a mindless change I made without too much consideration.

I suspect the 3 xfailed tests are a result of the embed being just ever-so-slightly different than originally expected, due to the API change you made with regards to what parameters are being passed via the `rates_dict`.

Reply:
After running the bot using [my own rates file](http://poglink-testing.s3-website.ca-central-1.amazonaws.com/)) I noticed that it was sending embeds correctly on the first run, but only sending default values for `name` and `color` when the rates changed.

`send_embed` is only called in two places, on initialization (if the `send_embed_on_startup` option is used), and when rates change 

Because we were only passing the `url` to send_embed() in this loop (`rates_dict={"url": url}`), the other keys were being set to their defaults. 

Updated this, now its 
```send_embed(
embed_description, rates_dict=self.rates_dicts[idx]
```

Reply:
Good catch, that was one of my hasty/lazy changes when clobbering to get tests executing successfully (not even passing, just not throwing uncaught exceptions haha). You should have seen the mess of a working directory I had in my git.

I hope I didn't leave things in too messy of a state though

Reply:
I definitely left things in a messier state lol